server/routes/public.js

```diff
@@ -43,7 +43,7 @@ router.get('/create-user', async (req, res) => {
         schoolId: Joi.number().positive().integer(),
         userType: Joi.string().min(1).valid('student', 'worker'),
         type: Joi.string(),
-        specialization: Joi.string().valid('social worker', 'guidance councellor'),//verify options
+        specialization: Joi.string(),//verify options
```

Comment:
Updated backend code to ensure correct value was called for variable.

Reply:
Understood!